### ISSUE TITLE
Add tutorial to https://sparktutorials.github.io/

### ISSUE THREAD

**Issue by tipsy** [SeunMatt/cloudcontact#1]
Hi Seun,

To add the tutorial to https://sparktutorials.github.io/, you have to clone https://github.com/sparktutorials/sparktutorials.github.io, add the tutorial (copy the content of readme.md, and add some metadata like author, publication date, allowing comments, etc) and create a pull-request. 

Let me know if you need any help.

//david

**Comment by SeunMatt** [comment]
Yes sir,
I really need help
I have forked the repo, then create a new branch, then I push my local repo to the new branch.

But then clicking on create new pull request from my own forked repo with the new branch just seems not to be working. 
Where should I go from here?
Thanks.
Once again,
- I have forked https://github.com/sparktutorials/sparktutorials.github.io
- Created a branch I called cloudcontact_tutorial
- I have then push my local repo of cloudcontact to the newly created branch.

What am I doing wrong or am not doing at all? Kindly step-wisely enlighten me.

Thanks
